spec/cell_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rspec'
+require './lib/cell'
+require './lib/ship'
+
+RSpec.describe do
+  describe '#initilaize' do
+    it 'cell exists' do
+      cell = Cell.new("B4")
+
+      expect(cell).to be_a(Cell)
+    end
+    
+    it 'cell has coordinates' do
+      cell = Cell.new("B4")
+
+      expect(cell.coordinate).to be_a("B4")
+    end
+    
+    it 'cell has no ships' do
+      cell = Cell.new("B4")
+
+      expect(cell.ship).to be_a(nil)
+    end
+    
+    it 'cell is empty' do
```

Comment:
let's move this test down into the 'can hold ships?' area

Reply:
That's such a great idea!